SPECS/dracut/90livenet/liveos-artifacts-download.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=LiveOS Artifacts Download Daemon
+After=network-online.target
```

Comment:
If the service is started manually using `systemctl start` then there isn't much point to adding `After` and `Wants`.

Reply:
We start it early before udev and the network stack is up, and then it'll block until 'After' `network-online.target`.